.jules/ferris.md

```
@@ -0,0 +1,3 @@
+## 2024-05-24 - [Avoid unnecessary map with to_string() when iterating over &&str]
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)